pom.xml

```diff
@@ -387,7 +383,12 @@
             <dependency>
                 <groupId>com.google.code.gson</groupId>
                 <artifactId>gson</artifactId>
-                <version>2.8.6</version>
+                <version>2.8.7</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jenkins-ci.main</groupId>
+                <artifactId>jenkins-test-harness</artifactId>
+                <version>1736.vc72c458c5103</version>
```

Comment:
nit: Could the parent pom be updated instead? Also for what it's worth you can set the property `<jenkins-test-harness.version>` without having to declare the whole dependency.